### PR TITLE
[TW-1317] Partner Workspace updates

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* In `roles-and-permissions.md`, updated "Partner roles" to "Team and Partner Workspace roles" to clarify what the roles relate to.
  *  Added mentions of the workspace and collection level roles you can assign to partners, and clarify the different permissions.
  * [TW-1349]: Clarify that you can assign the Partner Manager role at the team level, but the Partner role can only be assigned by inviting a partner. Also clarified that you can change partner permissions at the workspace and collection levels.
* In `partner-workspaces.md`, removed role descriptions from "Collaborating as a partner" and "Managing users and roles". Instead each sections point to `roles-and-permissions.md` for details about role permissions.
  * Added a new section titled "Adding partner licenses" that elaborates on consuming paid seats.
* Updated additional links and wording as needed.

[TW-1349]: https://postmanlabs.atlassian.net/browse/TW-1349?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

***
Beta site: [Partner Workspaces](https://tw-1317-partner-workspace-editor-viewer-updates.learning.postman-beta.com/docs/collaborating-in-postman/using-workspaces/partner-workspaces/), [Partner team and Partner Workspace roles](https://tw-1317-partner-workspace-editor-viewer-updates.learning.postman-beta.com/docs/collaborating-in-postman/roles-and-permissions/#partner-team-and-partner-workspace-roles)

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md [hunk 1]
```diff
@@ -151,44 +150,33 @@ The collaborative experience that Partner Workspaces offer depends on your role
...
+At the team level, all external partners are automatically assigned the Partner role. At the workspace level, all partners are assigned the [Workspace Editor or Viewer](/docs/collaborating-in-postman/roles-and-permissions/#partner-team-and-partner-workspace-roles) roles. If partners are assigned the Workspace Editor role, they can edit all workspace resources in the Partner Workspace, including collections. If partners are assigned the Workspace Viewer role, they can be assigned the Collection Editor role in specific collections. Partners can also be assigned the Partner Lead role, which enables them to invite other individuals from their organization to the Partner Workspace.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:159]: > If partners are assigned the Workspace Viewer role, they can be assigned the Collection Editor role in specific collections.

I think "for" or "on" might work better than "in" here

**Review by claireriley — APPROVED** [review]
Looks great! A major improvement 🌟

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md (hunk 1 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:159]: I see what you're saying. I can update this. I'll look for other/similar occurrences as well. Thanks!

**Review by akshay-kulkarni-03 — APPROVED** [review]

**Review by claireriley — APPROVED** [review]
LGTM!

**Pull request merged**